gen_cert.py

```diff
@@ -135,7 +138,14 @@ def __init__(self, course_id, template_pdf=None, aws_id=None, aws_key=None,
         self.aws_id = str(aws_id)
         self.aws_key = str(aws_key)
 
+        self.course_id = course_id
+        self.designation = designation
```

Comment:
Nit (optional): If you moved these lines down one line instead,
you'd have one solid block of added code, instead of two smaller ones,
as it doesn't appear they are particularly order-dependent.